board/safety/safety_honda.h

```diff
@@ -8,7 +8,7 @@
 //      brake > 0mph
 
 // these are set in the Honda safety hooks...this is the wrong place
-const int gas_interceptor_threshold = 328;
+const int gas_interceptor_threshold = 800;
```

Comment:
As we were discussing on the other PR, this should be 120/0.2539 (+1 to be slightly conservative and not false trigger panda cancellations before openpilot does) in order to be consistent with opendbc and openpilot. If this threshold does not work, we should figure out why.

Reply:
I know that 600 does not work, so 472 would definitely not work.  If I set it higher, occasional momentary gas signal spikes (either a-to-d or physical pedal anomaly) will cause OP to throw a "controls mismatch" error.

Reply:
I should note that it does a good job knowing it should disengage when I press on the gas with these settings.

Reply:
"I don't know why it seems the dbc is more forgiving." <-- yeah... this needs to be figured out before we can merge changes.

Reply:
Could it have anything to do with the GAS2 value? If you divide 120 by 0.126992032, you get a much closer value of 944.94.